__tests__/end-to-end/01login.test.js

```diff
@@ -105,7 +105,7 @@ describe(requirement(3), () => {
   });
 });
 
-describe(requirement(4), () => {
+describe.skip(requirement(4), () => {
```

Comment:
Oiie, acho que esses subiram juntos, mas é bom tirar pois o Renato falou que pode interferir no avaliador do github
```suggestion
describe(requirement(4), () => {
```